plugins/contextPlugin/search_visualizer.tid

```diff
@@ -2,7 +2,16 @@ title: $:/plugins/danielo515/ContextPlugin/visualizer
 tags: $:/tags/SearchResults
 caption: Context
 
-<$list filter="[!is[system]search{$:/temp/search}sort[title]limit[250]]">
+\define searchResults()
+<$set name="resultCount" value="""<$count filter="[!is[system]search{$(searchTiddler)$}]"/>""">
+
+{{$:/language/Search/Matches/Standard/Matches}}
+
+</$set>
+<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]]">
   {{!!title||$:/core/ui/ListItemTemplate}}
-  <$context term={{$:/temp/search}} />
+  <$context term={{$(searchTiddler)$}} />
```

Comment:
Where is this `searchTiddlers` value coming from? I can't see a set or macro referencing it.

Reply:
Ok, I saw the docs from 2014. This plugin is from 2013 (wow, time flies) that's why I was not aware.